doc/en/releasing.md

```diff
@@ -1,12 +1,15 @@
-# Releasing a new verison of the upgrader
+# Releasing a new version of the upgrader
 
-This document is aimed for internal consumption. To release a new version of the upgrader follow these steps.
+This document is aimed for internal consumption. 
+
+Upgrader releases are now automated with a travis build. When the tag gets built on travis, the build script will automatically run and build a PHAR executable. The PHAR executable will be attached back to the release in GitHub and publish to this repo's github pages.
+
+
+To release a new version of the upgrader follow these steps.
 
 1. On your local system, clone the upgrader codebase. You need to work of the `master` branch from this point.
-2. Increment the version number in `src/upgrade-code`, commit your changes and push them back up to GitHub.
-3. Call `php build.php` on the command line. This will build the upgrader as a PHAR file. 
-4. Move `upgrade-code.phar` to a temporary folder.
-5. Switch to the `gh-pages` branch. e.g.: `git checkout gh-pages`
-6. Overwrite the existing `upgrade-code.phar` with the one you built in step 3.
-7. Commit your changes to the `gh-pages` branch and push them up to GitHub. 
-8. Create a new release on GitHub and attach `upgrade-code.phar` to it.
+2. Increment the version number in `src/upgrade-code`, commit your changes and push them back up to GitHub. This will only be used for people installing the upgrader with Composer. The PHAR executable gets its version number from an environment file built into it. 
```

Comment:
Did we ever get anywhere with the automated version number updating?

Reply:
It's automated for the PHAR build, but there's no easy way to automate it for people who install the upgrader with `composer global`.